desc/plotting.py

```diff
@@ -2085,7 +2092,21 @@ def plot_boundaries(
         fig, ax = plot_boundaries((eq1, eq2, eq3))
 
     """
+    # if NFPs are not all equal, means there are
+    # objects with differing NFPs, which it is not clear
+    # how to choose the phis for by default, so we will throw an error
+    # unless phi was given.
     phi = parse_argname_change(phi, kwargs, "zeta", "phi")
+    errorif(
+        not np.allclose([thing.NFP for thing in eqs], eqs[0].NFP) and phi is None,
```

Comment:
@dpanici Maybe we should change this to 
```python
not np.allclose([thing.NFP for thing in eqs], eqs[0].NFP) and (phi is None or isinstance(phi, numbers.Integral)),
```
Because integer `phi` is still ambigious.

Reply:
ah, what does it do rn if phi integral is passed in? btw 0->2pi or 0->NFP of the first thing?

Reply:
It uses 0 to 2pi/nfp of the last thing.

Reply:
Also, doesn't throw any error or warning.